test/specs/components/action.spec.jsx

```diff
@@ -3,18 +3,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import TestUtils from 'react-addons-test-utils';
 import deepAssign from 'deep-assign';
-
-import { mockAppStore } from 'test/utils/redux';
-import { mockComponent } from 'test/utils/react';
-
 import StripeCheckout from 'react-stripe-checkout';
 
-import { ActionComponent } from 'components/action.jsx';
-import { LoadingComponent } from 'components/loading.jsx';
+import { mockAppStore } from '../../utils/redux';
+import { mockComponent } from '../../utils/react';
+
+import { ActionComponent } from '../../../src/js/components/action.jsx';
+import { LoadingComponent } from '../../../src/js/components/loading.jsx';
 
-const stripeService = require('services/stripe-service');
-const userService = require('services/user-service');
-const conversationService = require('services/conversation-service');
+const stripeService = require('../../../src/js/services/stripe-service');
```

Comment:
I'd rather not using aliases at all... we don't add new test files everyday for the widget.. it's mostly a write once thing